examples/llm/src/models/mosaic_gpt.py

```diff
@@ -239,7 +239,7 @@ class ComposerMosaicGPT(ComposerModel):
     def __init__(self, cfg):
         super().__init__()
         self.model = MosaicGPT(cfg)
-        self.__num_fwd_flops = None
+        self.num_fwd_flops = self.compute_num_fwd_flops()
```

Comment:
```suggestion
        self.num_fwd_flops = self._compute_num_fwd_flops()
```
Nit -- make private function?